irods_docker_files/docker_cmds_utilities.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+from __future__ import print_function
+
+import time
+import subprocess
+from subprocess import Popen, PIPE
+
+
+def get_docker_cmd(run_cmd, exec_cmd, stop_cmd, container_name, alias_name, database_container, database_type, network_name, extra_args=None):
+    docker_cmd = {'run_cmd': run_cmd,
+                  'exec_cmd': exec_cmd,
+                  'stop_cmd': stop_cmd,
+                  'container_name': container_name,
+                  'alias_name': alias_name,
+                  'database_container': database_container,
+                  'database_type': database_type,
+                  'network_name': network_name
+                 }
+    if extra_args is not None:
+        docker_cmd.update(extra_args)
+
+    print(docker_cmd)
+    return docker_cmd
+
+def build_irods_zone(build_tag, base_image, database_type, dockerfile='Dockerfile.install_and_test', install_database=True):
+    docker_cmd =  ['docker build -t {0} --build-arg base_image={1} -f {2} .'.format(build_tag, base_image, dockerfile)]
+    run_build = subprocess.check_call(docker_cmd, shell = True)
+    if install_database:
+        if database_type == 'oracle':
+            docker_cmd = ['docker build -t {0} -f Dockerfile.xe .'.format('oracle/database:11.2.0.2-xe')]
+            run_build = subprocess.check_call(docker_cmd, shell = True)
+        else:
+            import configuration
+            database_image = configuration.database_dict[database_type]
+            pull_image = ['docker pull {0}'.format(database_image)]
```

Comment:
pull_image = ['docker', 'pull', database_image]

Reply:
will tweak these later - merging for now